spec/models/user_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  let(:user){User.new()}
+
+  describe 'user attributes' do
+    it 'has expected attributes' do
+      expect(user.attribute_names.map(&:to_sym)).to contain_exactly(
+        :created_at, 
+        :email, 
+        :id, 
+        :photo, 
```

Comment:
if an user profile auth provider returns doesn't have a photo we probably need to add a default link